src/js/trove/iolib.js

```diff
@@ -0,0 +1,51 @@
+({
+    requires: [],
+    provides: {
+      values: {
+        "prompt": "tany"
+      }
+    },
+    nativeRequires: ["fs"],
+    theModule: function(RUNTIME, NAMESPACE, uri, fs) {
+      // https://github.com/nodejs/node/issues/28243#issuecomment-502402453
+      function PromptIO(msg) {
```

Comment:
According to the original comment, you were considering `input` rather than `prompt`.  Wouldn't it be more orthogonal to implement `input`, and then implement `prompt` as a combination of print and input?

Reply:
I'm not sure I like the synchronous nature of this implementation.  Since Pyret has the ability to pause the Pyret stack while an async operation occurs, and then resume the Pyret code with the results of that stack, I think it'd probably be cleaner to implement this using async-IO and pausing/resuming Pyret.  @jpolitz what's your take?